lib/ansible/modules/cloud/amazon/ec2_eni.py

```diff
@@ -108,6 +108,9 @@
 extends_documentation_fragment:
     - aws
     - ec2
+notes:
+    - This modules leverage the uniqueness of end_id, private_ip_address and subnet_id,
```

Comment:
How about "This module identifies and ENI based on either the eni_id, a combination of private_ip_address and subnet_id, or a combination of instance_id and device_id. Any of these options will let you specify a particular ENI." ?